samples/Compute/platform-image-policy/azurepolicy.json

```diff
@@ -3,8 +3,21 @@
     "name": "platform-image-policy",
     "properties": {
         "displayName": "Only allow a certain VM platform image",
-        "description": "This policy ensures that only UbuntuServer, Canonical is allowed from the image repository",
-        "parameters": {},
+        "description": "This policy ensures that only certain VM platform images are allowed from the image repository",
+        "parameters": {
+            "imagePublishers": {
+                "type": "array",
+                "defaultValue": "Canonical"
+            },
+            "imageOffers": {
+                "type": "array",
+                "defaultValue": "UbuntuServer"
+            },
+            "imageSkus": {
+                "type": "array",
+                "defaultValue": "14.04.2-LTS"
```

Comment:
Please provide multiple values in the array for one of the parameters, so that the example is more robust. 

You can see all the available SKUs, for example, through the [VM create blade](https://ms.portal.azure.com/#blade/Microsoft_Azure_Marketplace/GalleryResultsListBlade/selectedSubMenuItemId/%7B%22menuItemId%22%3A%22groups%2FCanonical.UbuntuServer%22%2C%22resourceGroupId%22%3A%22%22%2C%22resourceGroupLocation%22%3A%22%22%2C%22dontDiscardJourney%22%3Atrue%2C%22launchingContext%22%3A%7B%22galleryItemId%22%3A%22Canonical.UbuntuServer%22%2C%22source%22%3A%5B%22Resources%20Microsoft.Compute%2FVirtualMachines%22%2C%22GalleryFeaturedMenuItemPart%22%5D%2C%22menuItemId%22%3A%22Compute_MP%22%2C%22subMenuItemId%22%3A%22recommended%22%7D%7D)